consensus/src/marshal/actor.rs

```diff
@@ -696,6 +713,14 @@ impl<B: Block, E: Rng + Spawner + Metrics + Clock + GClock + Storage, V: Variant
         let _ = notifier.try_send(());
     }
 
+    /// Get the tip of the finalized block sequence.
+    async fn get_tip(&mut self) -> Option<(u64, B::Commitment)> {
+        let height = self.finalizations_by_height.last_index()?;
```

Comment:
nit: Can you add a comment here that the tip will always be a finalization? I think that was a non-intuitive leap we made.